types/favico.js/index.d.ts

```diff
@@ -30,5 +31,6 @@ declare namespace favicojs {
         webcam(): void;
     }
 }
+declare module "favico.js";
```

Comment:
This doesn't seem like the thing to do; I think what should actually be happening here is to restructure this file as an AMD-style module, like:

```
declare namespace Favico {
   /// ...
}

export = Favico;
export as namespace Favico;